src/search/SearchMessagesScreen.js

```diff
@@ -1,30 +1,70 @@
 /* @flow strict-local */
 import React, { PureComponent } from 'react';
 
+import * as api from '../api';
+import type { Auth, Message } from '../types';
 import { Screen } from '../common';
 import SearchMessagesCard from './SearchMessagesCard';
 import styles from '../styles';
+import { SEARCH_NARROW } from '../utils/narrow';
+import { LAST_MESSAGE_ANCHOR } from '../constants';
+import { connect } from '../react-redux';
+import { getAuth } from '../account/accountsSelectors';
 
-type Props = {||};
+type Props = {|
+  auth: Auth,
+|};
```

Comment:
That... doesn't seem to be the problem that the `$FlowFixMe` is (currently?) suppressing? Flow produces identical error output with or without `dispatch: Dispatch,` after that `$FlowFixMe` is removed:

```
 • in property search.screen:
    • Either property navigationOptions is missing in React.ComponentType [1] but exists in
      withOptionalNavigationOptions [2].
    • Or React.ComponentType [1] is incompatible with React.StatelessFunctionalComponent [3].
```

I've added it anyway, though; `connect` [is documented as supplying it](https://react-redux.js.org/using-react-redux/connect-mapdispatch), and probably _should_ be complaining about it already at its call site in this file.

Reply:
Yeah, those nav-related errors are what the fixme is aimed at suppressing. I think the failure to give an error here isn't so much because of the fixme as because the `react-navigation` types are no good (i.e. for broadly the same reason the fixme is there.)

In particular I think if the `connect` result were actually used to instantiate a component, then Flow would notice the missing `dispatch`. Ditto if it were wired through some well-typed higher-order components and ultimately used to instantiate a component.

... Then again if I try it on a random other component (`UnreadCards`, the first I saw in a quick grep), deleting either `dispatch` or another `connect`-supplied property from the `Props` type gives an error right at the `connect` call site. ... Oho, but if I delete the one call site of that component -- even though that's off in a different file -- the error disappears. So yeah, that seems to be what's happening.

Reply:
The commit 776df74ff "search: enforce ordering on network requests" now deletes this line; I'm guessing that's a rebase error.